docs/benchmarks/glyco/make_truth.py

```
@@ -0,0 +1,191 @@
+    Two details that are easy to get wrong:
+      * `spectrumID` is an INDEX (`index=40`), not a scan number. The scan is in the
+        `spectrum title` cvParam, in the classic `file.scan.scan` form.
+      * the glycan appears as a modification MASS DELTA, not a composition, so tables from
+        this reader carry `glycan` as `mass:<delta>` and compare by mass rather than by
+        composition. That is a weaker comparison than pGlyco2 allows, and is labelled so.
+    """
+    import xml.etree.ElementTree as ET
+    out = []
+    for path in paths:
+        run = os.path.basename(path).replace(".raw.mzid", "").replace(".mzid", "")
+        root = ET.parse(path).getroot()
```

Ruff (0.16.3)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)